devices/evn.js

```diff
@@ -1,26 +1,31 @@
 const exposes = require('../lib/exposes');
+const extend = require('../lib/extend');
 const fz = {...require('../converters/fromZigbee'), legacy: require('../lib/legacy').fromZigbee};
-const reporting = require('../lib/reporting');
 const e = exposes.presets;
 
 module.exports = [
     {
-        zigbeeModel: ['EVN ZBHS4RGBW'],
+        zigbeeModel: ['ZBHS4RGBW'],
         model: 'ZBHS4RGBW',
         vendor: 'EVN',
         description: 'Zigbee 4 channel RGBW remote control',
-        fromZigbee: [fz.battery, fz.command_on, fz.command_off, fz.command_step, fz.command_move, fz.command_stop, fz.command_recall,
-            fz.command_move_hue, fz.command_move_to_color, fz.command_move_to_color_temp],
-        exposes: [e.battery(), e.action(['on', 'off', 'brightness_step_up', 'brightness_step_down', 'brightness_move_up',
-            'brightness_move_down', 'brightness_stop', 'recall_1', 'recall_2', 'recall_3', 'hue_move', 'color_temperature_move',
-            'color_move', 'hue_stop'])],
+        fromZigbee: [fz.battery, fz.command_move_to_color, fz.command_move_to_color_temp, fz.command_move_hue,
+            fz.command_step, fz.command_stop, fz.command_move, fz.command_recall, fz.command_on, fz.command_off],
+        exposes: [e.battery(), e.action([
+            'color_move', 'color_temperature_move', 'brightness_step_up', 'brightness_step_down',
+            'brightness_move_up', 'brightness_move_down', 'brightness_stop',
+            'hue_move', 'hue_stop', 'recall_*', 'on', 'off'])],
         toZigbee: [],
         meta: {multiEndpoint: true, battery: {dontDividePercentage: true}},
-        configure: async (device, coordinatorEndpoint) => {
```

Comment:
Does it still report battery after this is removed? Repair the device and check if it still does.

Reply:
I am not so familiar with the reporting mechanism. But after switching on the device, the following message appears:

`Received Zigbee message from 'RGBW Fernbedienung', type 'attributeReport', cluster 'genPowerCfg', data '{"batteryPercentageRemaining":100}' from endpoint 1 with groupID null`

After that, this message comes cyclically:

`MQTT publish: topic 'zigbee2mqtt/RGBW Fernbedienung', payload '{"battery":100,"last_seen":1648152834008,"linkquality":111,"voltage":3800}'`